js/app/modules/card/defaultFamily.js

```diff
@@ -192,21 +291,33 @@ const Thumb = new Module.Class({
     vfunc_size_allocate: function (alloc) {
         this._layout.clear_constraints();
```

Comment:
just a reminder: don't _always_ remove the constraints, move all this logic to a separate method and replace constraints only when really necessary... otherwise the performance cost is really high because the solver has a to start from scratch every time (instead of reusing previous calculations).